build.cmd

```diff
@@ -1,3 +1,3 @@
 @echo off
-powershell -NoLogo -NoProfile -ExecutionPolicy ByPass -Command "& """%~dp0eng\common\build.ps1""" -build -restore -p:Projects="""%~dp0MSBuild.sln""" %*"
+powershell -NoLogo -NoProfile -ExecutionPolicy ByPass -Command "& """%~dp0eng\common\build.ps1""" -build -restore /p:Projects="""%~dp0MSBuild.sln""" %*"
```

Comment:
Was this really necessary? MSBuild should have supported both; is it a powershell problem?

Reply:
Powershell complained about it. Not a msbuild problem.